source/core/CEntryPoints.cpp

```diff
@@ -154,6 +154,36 @@ VIREO_EXPORT EggShellResult EggShell_DeallocateData(TypeManagerRef tm, const Typ
 
     return kEggShellResult_Success;
 }
+
+//------------------------------------------------------------
+//! ReinitializeToDefault
```

Comment:
Please update the comment to something likes this:
Clears data and reinitializes to default in dataRef described by typeRef

Reply:
Done.